java/mykola-melnyk/src/com/zoolatech/lecture1/tasks/_8/MainV2.java

```diff
@@ -0,0 +1,31 @@
+package com.zoolatech.lecture1.tasks._8;
+
+/**
+ * Write a program that prints all numbers from 1 to N, that are divisible by 2, 3 or by both.
+ * If a number is divisible by only 2 - print it and add in parentheses “(by 2)”,
+ * if only by 3 - “(by 3)”, if by both numbers - “(by 2 and 3)”. Every number should be printed
+ * on the new line. Create two versions of the program: one should use the for loop, second - the while loop.
+ */
+
+import java.util.Scanner;
+
+public class MainV2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input a number:");
+        int num = scanner.nextInt();
+        int start = 1;
+        while (start <= num) {
+
```

Comment:
extra new lines here might decrease the code readability

Reply:
Resolved: removed extra new lines.